content/BatchRequestContent.md

```diff
@@ -21,7 +22,34 @@ A batch cannot hold more that 20 requests and requests cannot contain a batch re
 
 Requests that require additional consent, not already obtained, should not be included in a batch.
 
+## Usage Examples
+
+```cs
+// create the batch request
+var batchRequestContent = new BatchRequestContent(graphServiceClient);
+var requestInformation = graphServiceClient.Users.ToGetRequestInformation();
+
+// add step using requestInformation
+var requestStepId = await batchRequestContent.AddBatchRequestStepAsync(requestInformation);
+
+// add step using native platform HTTP request
+var httpRequestMessage = new HttpRequestMessage(HttpMethod.Get, "http://example.com");
+var secondRequestStepId = batchRequestContent.AddBatchRequestStep(httpRequestMessage);
+
+// send and get back response
+var batchResponseContent = await graphServiceClient.Batch.PostAsync(batchRequestContent);
+
+// enumerate list of failed requests and create a new batch request
+var statusCodes = await batchResponseContent.GetResponsesStatusCodesAsync();
+if(statusCodes.Any())
+{
+    var retryBatch = batchRequestContent.NewBatchWithFailedRequests(rateLimitedResponses);
```

Comment:
Where is rateLimitedResponses coming from?

Reply:
Updated/fixed the sample.